BBC-archive/psammead
# Update @storybook/react from 4.0.0-alpha.8 to ^4.0.11

Resolves #128

Update to use stable version of Storybook React `4.0.11`, rather than the alpha version `4.0.0-alpha.8`. 

**Testing instructions**

```
git checkout storybook-fix-use-latest
npm ci
npm run storybook
```
Visit storybook at http://localhost8080
Components should be visible, with the relevant styles. Note: Reith font and normalise are not yet set up for Storybook. This will be done in a later PR. 

- [ ] ~Tests added for new features~ N/A
